core/sparqlbuilder/src/test/java/org/eclipse/rdf4j/sparqlbuilder/examples/updatespec/Section3Test.java

```diff
@@ -342,4 +342,30 @@ public void example_14() {
 	public void example_15() {
 		p(Queries.ADD().fromDefault().to(iri("http://example.org/named")));
 	}
+
+	/**
+	 * DELETE { GRAPH ?g1 { ?subject <http://my-example.com/anyIRI/> ?object . ?subject ?predicate
+	 * <http://my-example.com/anyIRI/> . } } WHERE { GRAPH ?g1 { OPTIONAL { ?subject ?predicate
+	 * <http://my-example.com/anyIRI/> . } OPTIONAL { ?subject <http://my-example.com/anyIRI/> ?object . } } }
+	 */
+	@Test
+	public void example_issue_1481() {
```

Comment:
This has been copied to a new class ModifyQueryTest, which is good, but you still need to remove it from Section3Test again.

Reply:
We have made removed the function from Section3Test